config/stage.json

```diff
@@ -0,0 +1,10 @@
+{
+    "EncryptorContainer" : {
+    "defaultEncryptor" : "AwsPrivateKeyDBFieldEncryption"
+    },
+    "isDeployedStage" : true, 
+    "rdsAwsSecretKey" : "ENGINE_RDS_STAGE",
```

Comment:
rdsAwsSecretKey and masterPrivateKeySecretName, etc, are always ENGINE_RDS_{ENV}, etc. So these I think we can determine programmatically based on ENV instead of setting in each properties file.